src/interface/others/RestorationShamanSpreadsheet.js

```diff
@@ -19,7 +19,7 @@ class RestorationShamanSpreadsheet extends React.Component {
 
     const getAbility = spellId => parser._modules.abilityTracker.getAbility(spellId);
     const casts = spellId => getAbility(spellId).casts;
-    const cpm = spellId => casts(spellId) / (parser.fightDuration / 1000 / 60) >= 0 ? (casts(spellId) / (parser.fightDuration / 1000 / 60)).toFixed(2) : '0';
+    const cpm = (spellId, tw = false) => (tw ? getAbility(spellId).healingTwHits : casts(spellId)) / (parser.fightDuration / 1000 / 60) >= 0 ? ((tw ? getAbility(spellId).healingTwHits : casts(spellId)) / (parser.fightDuration / 1000 / 60)).toFixed(2) : '0';
```

Comment:
this would be much more readable as a standalone function. nested ternaries are no me gusta